lib/engine/action/payoff_all_debt.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
```

Comment:
this doesn't seem necessary

Reply:
i wonder if you accomplish this with special buy instead

Reply:
I could add 1 or 1 for Each player loan to the game which is just a placeholder for the normal payyoff loan action

Reply:
not sure what that means but if it's simpler sure

Reply:
I went with the special buy way. That worked really well